web/packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -368,6 +374,45 @@ export class DocumentsService {
     });
   }
 
+  refreshPtyTitle(
+    uri: DocumentUri,
+    {
+      shell,
+      cwd,
+      clusterName,
+      runtimeSettings,
+    }: {
+      shell: Shell;
+      cwd: string;
+      clusterName: string;
+      runtimeSettings: Pick<RuntimeSettings, 'platform' | 'defaultOsShellId'>;
+    }
+  ) {
+    const doc = this.getDocument(uri);
+    if (!doc) {
+      return;
+    }
+    const omitShellName =
+      runtimeSettings.platform === 'linux' ||
+      (runtimeSettings.platform === 'darwin' &&
+        shell.id === runtimeSettings.defaultOsShellId);
```

Comment:
Isn't this supposed to be `(linux || darwin) && usesDefaultOsShell`?

Reply:
Oh right, fixed.